File: app/src/main/java/kr/com/misemung/ui/DustContract.java
package kr.com.misemung.ui;

import kr.com.misemung.vo.AirInfo;

public class DustContract {

    interface View {

        void showDustResult(AirInfo airInfo, String name);

        void reload(AirInfo airInfo, String name);
    }

    interface UserActionsListener {
        void loadFineDustData(AirInfo airInfo, String name);
    }
}

File: app/src/main/java/kr/com/misemung/ui/DustContract.kt
package kr.com.misemung.ui

import kr.com.misemung.vo.AirInfo

class DustContract {
    interface View {
        fun showDustResult(airInfo: AirInfo?, name: String?)
        fun reload(airInfo: AirInfo?, name: String?)
    }

    internal interface UserActionsListener {
        fun loadFineDustData(airInfo: AirInfo?, name: String?)
    }
}